Taskfile.yml

```diff
@@ -45,15 +45,15 @@ tasks:
     desc: Run all CI checks.
     cmds: 
     - task: generator:ci
-    - task: controller:generate-crds # required here to prevent it running twice in controller:ci, remove once go-task dedupes dependencies
+    - task: controller:generate-kustomize # required here to prevent it running twice in controller:ci, remove once go-task dedupes dependencies
```

Comment:
Doesn't **task** dedup now? See PR #1690

Reply:
Yeah, had conflicts with that. Now fixed.